backend-project/small_eod/search/grammar.py

```diff
@@ -90,13 +91,16 @@ def __eq__(self, other):
 # the grammar
 AND, OR, NOT = map(CaselessLiteral, "AND OR NOT".split())
 
-searchTerm = Word(alphanums) | quotedString.setParseAction(removeQuotes)
+searchTerm = Word(printables, excludeChars=excludedChars) | quotedString.setParseAction(
```

Comment:
Mam prośbę, czy możesz dodać pozytywny i negatywny przypadek do testów wskazujący na okoliczności zmiany? Parser to coś co może potencjalnie w projekcie stanowić wyzwanie w utrzymaniu, więc być krytyczne, więc dobrze mieć go dobrze pokryty testami. Jednocześnie taki test – zważywszy na to, że są podobne testy – prawdopodobnie nie jest trudno dodać. Teraz 5 minut, za dwa lata godzina.